c2corg_api/views/user.py

```diff
@@ -116,20 +120,128 @@ def post(self):
             log.warning('Error persisting user', exc_info=True)
             raise HTTPInternalServerError('Error persisting user')
 
-        # also create a version for the profile
-        DocumentRest.create_new_version(user.profile, user.id)
```

Comment:
Any reason you moved the creation of the versions for the profile to the confirmation of the registration? To make it easier to delete un-confirmed users?